verification/curator-service/ui/src/components/common-form-fields/FormikFields.tsx

```diff
@@ -205,14 +204,18 @@ export function RequiredHelperText(
     const { values, touched } = useFormikContext<CaseFormValues>();
     return (
         <div>
-            {hasKey(touched, props.name) &&
-                touched[props.name] &&
-                hasKey(values, props.name) &&
-                (values[props.name] === undefined ||
-                    values[props.name] === null ||
-                    values[props.name] === '') && (
-                    <FormHelperText error>Required field</FormHelperText>
-                )}
+            <FormHelperText
+                error={
+                    hasKey(touched, props.name) &&
+                    touched[props.name] &&
+                    hasKey(values, props.name) &&
+                    (values[props.name] === undefined ||
+                        values[props.name] === null ||
+                        values[props.name] === '')
+                }
+            >
+                Required
```

Comment:
Can you remind me what the distinction is between the function of this text, and the Yup `required` string?

Reply:
I don't think the Yup required string is technically used anywhere yet. Formik keeps track of all the form errors in an object called errors. The Yup required string dictates what shows up in that Formik object. Though that isn't shown in the UI anywhere currently, it may be useful in the future and is also useful when debugging (I frequently print the contents of Formik's `error` to see what's going on in the form).